src/tscache.ts

```diff
@@ -118,6 +118,7 @@ export class TsCache
 
 		if (noCache)
 		{
+			this.context.warn(yellow("Cleaning cache... If you want to use caching, please set `clean` option to false."));
```

Comment:
```suggestion
```

The correct location for the logging is in `index.ts`. You added that in the most recent commit, so can remove this